### DISCUSSION TITLE
去除视频链接多余跟踪参数

### DISCUSSION THREAD

**Discussion by Kennnnnnji** [the1812/Bilibili-Evolved#1300] [提问 / Q&A] [answered]
这个功能非常好用，请问有可能做YouTube版本吗？
（如果没有时间和意愿做的话请问大佬能提供一下技术细节，怎么实现的吗？）

**Comment by the1812** [comment, marked as answer]
我们只会提供用于 b 站的功能.

源代码位于 https://github.com/the1812/Bilibili-Evolved/blob/master/src/utils/url-params-clean.ts
清理好链接里的参数用`history.replaceState({}, document.title, newUrl)`替换就行了
